api/turing/cluster/servicebuilder/router_test.go

```diff
@@ -64,7 +64,7 @@ func TestNewRouterService(t *testing.T) {
 	// Define tests
 	tests := map[string]testSuiteNewService{
 		"success | basic": {
-			filePath:     filepath.Join(testDataBasePath, "router_version_basic.json"),
```

Comment:
Is there any reason to modify existing test instead of adding new set of test for upi?

Reply:
most of other test cases already covered the http service building logic, I try to refrain from adding more json config tests to keep it leaner as its pretty hard to work with the current size of files and there are a lot of object construct in place here, hence the decision to turn use it as the grpc basic. 

No strong opinion, could always add another one 😅

Reply:
I think it's better to add new one.